lua/plenary/path.lua

```diff
@@ -20,8 +20,13 @@ path.home = vim.loop.os_homedir()
 
 path.sep = (function()
   if jit then
+    local shellslash_exists, _ = pcall(function() local _ = vim.o.shellslash end)
```

Comment:
I think this might be a more straightforward way to check for the `shellslash` option:

```suggestion
    local shellslash_exists = vim.fn.exists("+shellslash") ~= 0
```

Reply:
Yes that's better, thank you! I was trying to use that function but wasn't able to make it work (i missed the "+")